templates/kubernetes/compare.html

```diff
@@ -16,494 +20,502 @@
+                <p>
+                  <a href="https://assets.ubuntu.com/v1/b5f9ae49-Enterprise_Kubernetes_Datasheet.pdf">Kubernetes for the enterprise</a>
+                </p>
+              </div>
+            </div>
+          </li>
+          <li class="p-list__item">
+            <div class="row">
+              <div class="col-2">
+                <h3 class="p-heading--5">Webinar on demand</h3>
+              </div>
+              <div class="col-4">
+                <p>
+                  <a href="https://www.brighttalk.com/webcast/6793/435999">How to accelerate Kubernetes deployment in the enterprise</a>
+                </p>
+              </div>
+            </div>
+          </li>
+          <li class="p-list__item">
+            <div class="row">
+              <div class="col-2">
+                <h3 class="p-heading--5">Case study</h3>
+              </div>
+              <div class="col-4">
+                <p>
+                  <a href="/engage/atresmedia-charmed-kubernetes-case-study">Atresmedia dominates Spain's OTT media market with Charmed Kubernetes</a>
+                </p>
+              </div>
+            </div>
+          </li>
+        </ul>
+        <div class="p-cta-block">
+          <a href="/kubernetes/resources">Find more resources&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section--deep">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
         <h2>Need help with K8s?</h2>
+      </div>
+      <div class="col">
         <p>Let our expert cloud team help with:</p>
-        <ul class="p-list">
+        <ul class="p-list--divided">
           <li class="p-list__item is-ticked">Project evaluation</li>
           <li class="p-list__item is-ticked">Requirements and feature parity</li>
           <li class="p-list__item is-ticked">Pricing and TCO assessment</li>
         </ul>
-        <p>
+        <div class="p-cta-block">
           <a href="/kubernetes/contact-us"
              class="p-button--positive js-invoke-modal">Contact us</a>
-        </p>
-          </div>
         </div>
-      </section>
+      </div>
+    </div>
+  </section>
 
-      <!-- Set default Marketo information for contact form below-->
-      <div class="u-hide"
-           id="contact-form-container"
-           data-form-location="/shared/forms/interactive/kubernetes"
-           data-form-id="3230"
-           data-lp-id="6274"
-           data-return-url="https://ubuntu.com/kubernetes/thank-you?product=kubernetes-compare"
-           data-lp-url="https://pages.ubuntu.com/things-contact-us.html"></div>
+  <!-- Set default Marketo information for contact form below-->
+  <div class="u-hide"
+       id="contact-form-container"
+       data-form-location="/shared/forms/interactive/kubernetes"
+       data-form-id="3230"
+       data-lp-id="6274"
+       data-return-url="https://ubuntu.com/kubernetes/thank-you?product=kubernetes-compare"
```

Comment:
I may be wrong but are we not removing thank-you pages and replacing them with the #success tags.
Like this one https://ubuntu-com-14660.demos.haus/kubernetes/compare#success

Reply:
It actually routes to the banner as is but indeed probably better to explicit 🙂